susemanager-utils/susemanager-sls/src/modules/mgractionchains.py

```diff
@@ -160,6 +156,11 @@ def next(actionchain_id, chunk, next_action_id=None, ssh_extra_filerefs=None):
         yaml_dict['ssh_extra_filerefs'] = ssh_extra_filerefs
     _persist_next_ac_chunk(yaml_dict)
 
+def reboot(actionchain_id, chunk, next_action_id=None, ssh_extra_filerefs=None):
+    ret = __salt__['rebootmgr.reboot']()
```

Comment:
We can't assume that `rebootmgr` is used, in fact we'll probably be setting SLE Micro clients to not use `rebootmgr` and instead use `systemd` as the 'reboot manager' as discussed in https://github.com/uyuni-project/uyuni-rfc/pull/74.

To make this generic, this should just be 
```suggestion
    ret = __salt__['transactional_update.reboot']()
```

Reply:
oh actually with the new implementation this function in not call at all. I'll remove it.